documentation/staging/content/userguide/platforms/environments.md

```diff
@@ -121,6 +180,22 @@ AKS support and limitations:
 
 See also the [Azure Kubernetes Service sample]({{<relref "/samples/azure-kubernetes-service/_index.md">}}).
 
+##### Oracle WebLogic Server on the Azure Kubernetes Service Marketplace (WLS on AKS Marketplace)
+
+The WebLogic Server on AKS Marketplace
+offer lets you embrace cloud computing by providing greater choice
```

Comment:
How about

The WebLogic Server on AKS Azure Marketplace offer lets you...

Reply:
Fixed. Please resolve if it looks fine now

Reply:
Resolved.